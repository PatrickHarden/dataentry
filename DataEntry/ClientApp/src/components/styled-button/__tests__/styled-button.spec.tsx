import React from 'react';
import { mount, shallow } from 'enzyme';

import StyledButton from '../styled-button';

describe('components', () => {

  describe('styled-button', () => {

    let props:any;
    

    beforeEach(() => {
        props = {
            buttonStyle: '1',
            primary: false
        }
    });

    

    // snapshot render
    it('renders according to snapshot', () => {
        expect(shallow(<StyledButton {...props} />)).toMatchSnapshot();
    });
  });
});