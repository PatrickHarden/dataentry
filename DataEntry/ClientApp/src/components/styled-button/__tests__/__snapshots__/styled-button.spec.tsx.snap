// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components styled-button renders according to snapshot 1`] = `
<styled.button
  buttonStyle="1"
  onKeyPress={[Function]}
  primary={false}
  tabIndex={0}
/>
`;
